types/bool.go

```diff
@@ -12,37 +12,113 @@ var (
 	_ attr.Value = Bool{}
 )
 
+// BoolValue creates a Bool with a known value. Access the value via the Bool
+// type BoolValue method.
+//
+// Setting the deprecated Bool type Null, Unknown, or Value fields after
+// creating a Bool with this function has no effect.
+func BoolValue(value bool) Bool {
```

Comment:
Wonder whether we might want to consider using `ValueBool` for consistency with `NullBool` and `UnknownBool`? This would also differentiate the function for instantiating a `Bool` with a value from the function that retrieves the value (`b.BoolValue`).

Reply:
I'm okay with that. 👍

Reply:
Pushed up that change and rebased. 😄